tianshou/data/collector.py

```diff
@@ -242,18 +236,168 @@ def reset_buffer(self, keep_statistics: bool = False) -> None:
     def reset_env(
         self,
         gym_reset_kwargs: dict[str, Any] | None = None,
-    ) -> None:
+    ) -> tuple[np.ndarray, np.ndarray]:
```

Comment:
Return value should be documented in docstring.

Reply:
Done